### ISSUE TITLE
'use_WaveSF' is missed in the Configuration File

### ISSUE THREAD

**Issue by cyinen** [CyberPegasus/IFDD#2]
I executed the command "python tool/train.py" but encountered errors, likely due to missing parameters in the configuration file.

<img width="619" alt="image" src="https://github.com/user-attachments/assets/2119d459-85c6-4119-9d92-b9cc5fc63505" />

**Comment by CyberPegasus** [comment]
Thanks for your feedback. 
I missing them during code cleaning process.
I have updated them by modifying _dynamic_update function in configuration files `exps/EXP_IFDD_xxx.py`.

**Comment by cyinen** [comment]
你好，我使用了新上传的文件可以跑通了。但是使用默认的config，模型似乎无法收敛，loss没有下降。请问是哪里需要调整吗？还是要用预训练权重初始化？
<img width="1277" alt="image" src="https://github.com/user-attachments/assets/09554ecb-6575-463f-b544-2f265bd2ad54" />

**Comment by CyberPegasus** [comment]
Thanks for your interest in our work.

For DFEW, to provide a stable initial weights, it is suggested to pretrain MFViT backbone (created by function `MFViT_build_backbone` ) with detection head on DFEW dataset following contrastive learning setting. 
It involves a dataset class for contrastive learning, like `FERV39k_SSL` in `FERV39k.py`, and a loss function for contrastive learning , i.e., `SimCLRLoss` in `loss_SSL.py`.

Besides, higher performance can be achieved by pre-loading weights from MViT-S [http://arxiv.org/abs/2112.01526] to model.embed and model.backbone.backbone .

Please note that the code currently uploaded in this repository is not the final version.. 
As I plan to extend it to support additional video tasks, the complete code will be made available at a later time.

**Comment by CyberPegasus** [comment]
Besides, higher performance can be achieved by loading ViT weights to model.embed and model.backbone.backbone from MViT-S [http://arxiv.org/abs/2112.01526].
